app/models/user.rb

```diff
@@ -10,4 +12,38 @@ class User < ApplicationRecord
   validates :password, presence: true,
             length: {minimum: Settings.digits.length_6}
   has_secure_password
+
+  class << self
+    # Returns the hash digest of the given string.
+    def digest string
+      cost = if ActiveModel::SecurePassword.min_cost
+               BCrypt::Engine::MIN_COST
+             else
+               BCrypt::Engine.cost
+             end
+      BCrypt::Password.create(string, cost:)
+    end
+
+    # Returns a random token.
+    def new_token
+      SecureRandom.urlsafe_base64
+    end
+  end
+  # Remembers a user in the database for use in persistent sessions.
+  def remember
+    self.remember_token = User.new_token
+    update_attribute :remember_digest, User.digest(remember_token)
```

Comment:
```suggestion
    update_column :remember_digest, User.digest(remember_token)
```
ri mới là tốt nhất cho trường hợp này nhé